src/deviceApiCalls/schemas/getIncontextSignupDismissedAt.result.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "GetIncontextSignupDismissedAt",
+    "type": "object",
+    "description": "Gets the time that the in-context Email Protection sign-up message was dismissed, if set",
+    "additionalProperties": false,
+    "properties": {
+        "success": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "value": {
+                    "type": "number"
```

Comment:
Are we sure we want a number here? I understand this is a timestamp, so it should be ok. It's just that any time I've used a number in these schemas I always ended up reverting to a string after some time, for one reason or another. Maybe it's just me 🙃. Anyway, we may want to change the name `value` to `timestamp` to make it more explicit 😉.

Reply:
@GioSensation It may depend on how the backing datastore deals with eg (e.g. saving to localstorage would return you a string). In this case the Chrome browser storage retains the type, so we get a number back.
I'll update this to timestamp, but let me know if you'd be more comfortable with this dealing in strings.